Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -334,7 +360,46 @@ def run_checks(
           next info
         end
 
-        print_latest_version(info, verbose: verbose, ambiguous_cask: ambiguous_casks.include?(formula_or_cask))
+        print_latest_version(info, verbose: verbose, ambiguous_cask: ambiguous_casks.include?(formula_or_cask), check_resource: false)
```

Comment:
Up to this point, the output from `#print_latest_version` has come directly after the debug output without any separation:

```
Matched Versions:
1.8.0, 0.0.1, 0.0.2, 0.0.3, 0.0.4, 0.0.5, 0.0.6, 0.0.7, 0.0.8, 0.0.9, 0.1.0, ..., 2.0.0, 2.0.1, 2.0.2, 2.0.3, 2.0.4, 2.0.5, 2.0.6, 2.0.7, 2.0.8, 2.0.9, 2.1.0, 2.1.1, 2.2.0, 2.3.0
influxdb: 2.3.0 ==> 2.3.0
```

However, now that we may be printing multiple lines when checking resource versions, this can start to look a bit cramped:

```
...

Resource:          pkg-config-wrapper
Livecheckable?:    Yes

URL:              https://github.com/influxdata/pkg-config/tags
Strategy:         PageMatch
Regex:            /href="\/influxdata\/pkg-config\/archive\/refs\/tags\/v(.*).tar.gz"/i

Matched Versions:
0.2.12, 0.2.11, 0.2.10, 0.2.9, 0.2.8, 0.2.7, 0.2.6, 0.2.5, 0.2.4, 0.2.3

Resource:          ui-assets
Livecheckable?:    No
influxdb: 2.3.0 ==> 2.3.0
  pkg-config-wrapper: 0.2.11 ==> 0.2.12
  ui-assets: 2.1.2 ==> 2.1.2
```

There may be something to be said for printing a blank line before the output from `#print_latest_version` in the debug output:

```suggestion
        puts if debug
        print_latest_version(info, verbose: verbose, ambiguous_cask: ambiguous_casks.include?(formula_or_cask), check_resource: false)
```

This ends up looking like:

```
...

Resource:          ui-assets
Livecheckable?:    No

influxdb: 2.3.0 ==> 2.3.0
  pkg-config-wrapper: 0.2.11 ==> 0.2.12
  ui-assets: 2.1.2 ==> 2.1.2
```

Reply:
Totally agreed! It was missing for Formulae/Cask as well. Initially, I added a blank space there, but later removed it in order to not change the debug output for Formulae/Cask.